### PR TITLE
Changed Arguments to match the notation

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/21460

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details: 
   - [X] No

### CODE REVIEW

File: Integrations/FeedTAXII/FeedTAXII.py [hunk 1]
```diff
@@ -498,12 +503,12 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
         self.collection = collection
 
         # authentication
-        self.api_key = api_key
-        self.api_header = api_header
-        if not credentials:
-            credentials = {}
-        self.username = credentials.get('identifier', None)
-        self.password = credentials.get('password', None)
+        if '_header' in credentials.get('identifier', None):
```
Comment (ronykoz) [inline Integrations/FeedTAXII/FeedTAXII.py]:
```suggestion
        if '_header:' in credentials.get('identifier', None):
```

File: Integrations/FeedTAXII/FeedTAXII.py [hunk 2]
```diff
@@ -498,12 +503,12 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
...
+            self.api_key = credentials.get('identifier', None)
+            self.api_header = credentials.get('password', None)
```
Comment (ronykoz) [inline Integrations/FeedTAXII/FeedTAXII.py]:
```suggestion
            self.api_header = credentials.get('identifier', None).split('_header:')[1]
```

File: Integrations/FeedTAXII/FeedTAXII.py [hunk 3]
```diff
@@ -498,12 +503,12 @@ def __init__(self, insecure: bool = True, polling_timeout: int = 20, initial_int
...
 
         # authentication
-        self.api_key = api_key
-        self.api_header = api_header
-        if not credentials:
-            credentials = {}
-        self.username = credentials.get('identifier', None)
-        self.password = credentials.get('password', None)
+        if '_header' in credentials.get('identifier', None):
+            self.api_key = credentials.get('identifier', None)
```
Comment (ronykoz) [inline Integrations/FeedTAXII/FeedTAXII.py]:
```suggestion
            self.api_key = credentials.get('password', None)
```

**Comment by teizenman** [comment]
@kirbles19 Can you please go over the following changes:
1. Changelog: https://github.com/demisto/content/blob/taxii-credentials/Integrations/FeedTAXII/CHANGELOG.md
2. description: https://github.com/demisto/content/blob/taxii-credentials/Integrations/FeedTAXII/FeedTAXII_description.md

**Comment by kirbles19** [comment]
@teizenman @ronykoz Done.

**Pull request merged**